pandas_gbq/gbq.py

```diff
@@ -1001,15 +998,7 @@ def to_gbq(
     reauth : bool, default False
         Force Google BigQuery to re-authenticate the user. This is useful
         if multiple accounts are used.
-    if_exists : str, default 'fail'
```

Comment:
This is a breaking change, right?

I figured we could pretty easily map from `if_exists` to the corresponding write disposition without breaking our users.